### PR TITLE
Increase debug blocked reactor notify ms

### PR DESCRIPTION
**Pull request by bhalevy**
Timing in debug mode is much slower than release/dev modes and
the default reactor stall threshold of 200 ms produces
a lot of false positives. Therefore, increase it in this case using
the `--blocked-reactor-notify-ms` option to 5 seconds.

### CODE REVIEW

File: ccmlib/common.py [hunk 1]
```diff
@@ -399,20 +399,25 @@ def isOpscenter(install_dir):
     opscenter_script = os.path.join(bin_dir, 'opscenter')
     return os.path.exists(opscenter_script)
 
+def scylla_extract_mode(path):
+    m = re.match('(.*/)?build/(\w+)(/|$)', path)
```
Comment (fruch) [inline ccmlib/common.py]: can you add a few examples here ? i.e. one url one file path ?

Comment (bhalevy) [reply ccmlib/common.py]: Added in v2

**Comment by bhalevy** [comment]
In v2:
- extended scylla_extract_mode to support also `scylla(-{mode})?-package.tar.gz` urls.
- added path/url examples to scylla_extract_mode

**Comment by bhalevy** [comment]
BYO: https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/59
(with debug printouts: https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/60/consoleFull)

**Comment by bhalevy** [comment]
@scylladb/ccm-maint please merge

**Pull request merged**